awssqs/main.go

```diff
@@ -199,7 +199,7 @@ func (clients Clients) sendSQSEvent(msg *sqs.Message, queueARN *string) error {
 
 	event := cloudevents.Event{
 		Context: cloudevents.EventContextV03{
-			Type:            "AWS SQS Message",
+			Type:            "com.amazon.sqs",
```

Comment:
Ditto here. I think we need to extract something from the event itself - if possible.